terraform/variables.tf

```diff
@@ -43,6 +43,22 @@ variable "terraform_backend_dynamodb_lock_table" {
   description = "The DynamoDb table to store the Terraform state lock"
 }
 
+variable "terraform_backend_use_tfc" {
+  type        = bool
+  default     = false
+  description = "Whether to use Terraform Cloud as the remote backend. Defaults to false."
+}
+
+variable "terraform_backend_tfc_organization" {
+  type        = string
+  description = "The Terraform Cloud organization to use as the remote backend. Must be provided if terraform_backend_use_tfc is true."
+}
+
+variable "terraform_backend_tfc_workspace" {
+  type        = string
+  description = "The Terraform Cloud workspace to use as the remote backend. Must be provided if terraform_backend_use_tfc is true."
+}
```

Comment:
these aren't optional, which might cause some confusion. not sure if terragrunt has a good solution to that, though.

Reply:
ah you're right. I'll add defaults with empty strings for now